packages/pyright-internal/src/analyzer/typeEvaluator.ts

```diff
@@ -1334,11 +1334,7 @@ export function createTypeEvaluator(
     }
 
     function isAnnotationEvaluationPostponed(fileInfo: AnalyzerFileInfo) {
-        return (
-            fileInfo.futureImports.get('annotations') !== undefined ||
-            fileInfo.executionEnvironment.pythonVersion >= PythonVersion.V3_10 ||
```

Comment:
Should this be bumped to 3.11 instead, in hopes it's there?

Reply:
Yeah, I'm fine with that.